src/core/src/op/fake_convert.cpp

```diff
@@ -56,15 +54,21 @@ bool FakeConvert::visit_attributes(ov::AttributeVisitor& visitor) {
     return true;
 }
 
-void FakeConvert::validate() const {
-    OPENVINO_ASSERT(std::find(m_valid_types.begin(), m_valid_types.end(), m_destination_type) != m_valid_types.end(),
+void FakeConvert::validate_type() const {
+    const auto& valid_types = this->get_valid_types();
+    OPENVINO_ASSERT(std::find(valid_types.begin(), valid_types.end(), m_destination_type) != valid_types.end(),
                     "Bad format for f8 conversion type: " + m_destination_type);
 }
 
 bool FakeConvert::has_evaluate() const {
     return false;
 }
 
+const std::vector<std::string>& FakeConvert::get_valid_types() const {
```

Comment:
does it need to be a class method? looks like it can be a free function

Reply:
Agree, moved out of the class